spec/index.bs

```diff
@@ -1958,15 +1958,31 @@ interface GPUTextureUsage {
 
             **Arguments:**
             <pre class=argumentdef for="GPUDevice/createTexture(descriptor)">
-                descriptor: Description of the {{GPUTexture}} to create.
+                |descriptor|: Description of the {{GPUTexture}} to create.
             </pre>
 
             **Returns:** {{GPUTexture}}
 
-            Issue: Describe {{GPUDevice/createTexture()}} algorithm steps.
+            1. If any of the following conditions are unsatisfied, generate a validation
+                error and return an [=invalid=] {{GPUTexture}}.
+                <div class=validusage>
+                    - 1 &le; {{GPUTextureDescriptor/size}}.width
+                    - 1 &le; {{GPUTextureDescriptor/size}}.height
+                    - 1 &le; {{GPUTextureDescriptor/size}}.depth
+                    - Let |maxDim| = {{GPUTextureDescriptor/size}}.width
```

Comment:
aside: One reason I've been contemplating using "must" on every line is that it fixes the awkwardness of mixing boolean expressions with imperative statements like "Let x = ...". I don't know what the right style is though.

- 1 must &le; size.width
- 1 &le; size.width must be true
- ?